.gitignore

```diff
@@ -1,2 +1,3 @@
 __pycache__
-venv/
\ No newline at end of file
+venv/
+.idea/
```

Comment:
Try to add more files that come under the scope of Python projects. Refer `.gitignore` Python template

Reply:
Consider its done